components/eam/src/chemistry/utils/modal_aero_wateruptake.F90

```diff
@@ -336,15 +370,15 @@ subroutine modal_aero_wateruptake_dr(state, pbuf, list_idx_in, dgnumdry_m, dgnum
          end do ! i = 1, ncol
       end do ! k = top_lev, pver
 
-   end do ! m = 1, nmodes
+   end do    ! modes
 
    !----------------------------------------------------------------------------
    ! specify clear air relative humidity
 
    if (present(clear_rh_in)) then
 
-      ! use input relative humidity
-      rh(1:ncol,1:pver) = clear_rh_in(1:ncol,1:pver)
```

Comment:
Wait, what's going on here? Are you undoing this change I added for the MMF?

Reply:
Thanks @whannah1 for catching this. It got removed during cam->eam conflict resolution. I will reinstate this code and revisit this code again to ensure that there aren't anymore surprises.

Reply:
ah, I see, that's fine then.